tests/copyright/check_copyright.sh

```diff
@@ -65,6 +67,8 @@ ignore_pat="$ignore_pat|\\.qrc$"
 ignore_pat="$ignore_pat|/doc/devices\\.conf$"
 # Jenkins configuration
 ignore_pat="$ignore_pat|/Jenkinsfile"
+# upstart jobs
+ignore_pat="$ignore_pat|/plugins/.*\\.conf$"
```

Comment:
I guess we want to narrow this down to just the Wizard plugin. `.conf` is a somewhat generic extension.

```suggestion
ignore_pat="$ignore_pat|/plugins/Wizard/.*\\.conf$"
```

Reply:
OK.